src/pages/yeeun/Login/Login.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import './Login.scss';
+
+class Login extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      id: '',
+      pw: '',
+    };
+  }
+
+  goToMain = () => {
+    this.props.history.push('/main-yeeun');
+  };
+
+  handleIdInput = event => {
+    this.setState({
+      id: event.target.value,
+    });
+  };
+
+  handlePwInput = event => {
+    this.setState({
+      pw: event.target.value,
+    });
+  };
```

Comment:
```suggestion
  handleInputs = event => {
    this.setState({
      [event.target.name]: event.target.value,
    });
  };
```
위와 같이 id와 pw를 분리하지 않고 하나의 함수로 사용할 수 있습니다.
단, target이 되는 input에 name값을 각각 id와 pw로 지정해주셔야 합니다 :)